model/manifest.go

```diff
@@ -10,14 +11,61 @@ import (
 )
 
 type Manifest struct {
-	Id      string           `json:"id" yaml:"id"`
-	Backend *ManifestBackend `json:"backend,omitempty" yaml:"backend,omitempty"`
+	Id          string           `json:"id" yaml:"id"`
+	Name        string           `json:"name" yaml:"name"`
+	Description string           `json:"description" yaml:"description"`
+	Backend     *ManifestBackend `json:"backend,omitempty" yaml:"backend,omitempty"`
+	Webapp      *ManifestWebapp  `json:"webapp,omitempty" yaml:"webapp,omitempty"`
```

Comment:
How about making this field "frontend" to better complement "backend"?

Reply:
I prefer `Webapp` since we'll also likely have `ReactNative` and other clients in the future. We could toss them all under `Frontend` but that seems like unnecessary nesting to me

Reply:
👍